### PR TITLE
feat(ci): run apisix-seed as a docker

### LINKED ISSUE #31: test: e2e test should run apisix-seed as a docker

### PR DESCRIPTION
**Pull request by soulbird**
fix: #31

### CODE REVIEW

File: ci/apisix-seed/conf.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+etcd:
+  host:                           # it's possible to define multiple etcd hosts addresses of the same etcd cluster.
+    - "http://172.50.238.20:2379"     # multiple etcd address, if your etcd cluster enables TLS, please use https scheme,
```
Comment (shuaijinchao) [inline ci/apisix-seed/conf.yaml:3]: Why not use hostname?

File: ci/apisix-seed/conf.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    #cert: /path/to/cert          # path of certificate used by the etcd client
+    #key: /path/to/key            # path of key used by the etcd client
+
+    verify: true                  # whether to verify the etcd endpoint certificate when setup a TLS connection to etcd,
+    # the default value is true, e.g. the certificate will be verified strictly.
+
+discovery:                       # service discovery center
+  nacos:
+    host:                        # it's possible to define multiple nacos hosts addresses of the same nacos cluster.
+      - "http://172.50.238.30:8848"
```
Comment (shuaijinchao) [inline ci/apisix-seed/conf.yaml:19]: ditto

File: ci/apisix-seed/conf.yaml (hunk 1 above)
Comment (shuaijinchao) [inline ci/apisix-seed/conf.yaml:3]: Why not use hostname?

File: ci/apisix-seed/conf.yaml (hunk 2 above)
Comment (shuaijinchao) [inline ci/apisix-seed/conf.yaml:19]: ditto

File: ci/apisix-seed/conf.yaml (hunk 1 above)
Comment (soulbird) [reply ci/apisix-seed/conf.yaml:3]: DNS hasn't be supported, see #39

File: ci/docker-compose-arm64.yml [hunk 3]
```diff
@@ -45,6 +45,21 @@ services:
...
+  apisix_seed_dev:
+    build:
+      context: ./..
+      dockerfile: Dockerfile.local
+    restart: always
+    volumes:
+      - ./apisix-seed/conf.yaml:/usr/local/apisix-seed/conf/conf.yaml:ro
+    depends_on:
+      - apisix
+      - etcd
```
Comment (spacewander) [inline ci/docker-compose-arm64.yml:56]: We don't need to depend on etcd as apisix already does

Comment (soulbird) [reply ci/docker-compose-arm64.yml:56]: No, it will panic when etcd isn't running. But we can not depend on apisix

**Review by spacewander — APPROVED** [review]

**Pull request merged**